scripts/publish/bintray-debian.sh

```diff
@@ -37,29 +71,25 @@ if [ -z "$BINTRAY_USER" ] || [ -z "$BINTRAY_API_KEY" ]; then
 fi
 set -u
 
-ARGV_FILE=$1
-PACKAGE_COMPONENT=etcher
-PACKAGE_DISTRIBUTION=stable
-PACKAGE_FILE_NAME=$(basename $ARGV_FILE)
-PACKAGE_NAME=${PACKAGE_FILE_NAME%.*}
-PACKAGE_VERSION=$(echo $PACKAGE_NAME | cut -d_ -f2 | tr "~" "-")
-PACKAGE_ARCH=$(echo $PACKAGE_NAME | cut -d_ -f3)
-
-if [ -z $PACKAGE_VERSION ]; then
-  echo "Couldn't infer the version from the package file name" 1>&2
+if [ "$ARGV_RELEASE_TYPE" == "production" ]; then
+  PACKAGE_DISTRIBUTION=stable
+elif [ "$ARGV_RELEASE_TYPE" == "snapshot" ]; then
+  PACKAGE_DISTRIBUTION=devel
+else
+  echo "Invalid release type: $ARGV_RELEASE_TYPE" 1>&2
   exit 1
 fi
 
-if [ -z $PACKAGE_ARCH ]; then
-  echo "Couldn't infer the architecture from the package file name" 1>&2
-  exit 1
-fi
+PACKAGE_COMPONENT=$(echo "$ARGV_APPLICATION_NAME" | tr '[:upper:]' '[:lower:]')
```

Comment:
> Rather than doing the tr here, it's probably better to just pass in $(APPLICATION_NAME_LOWERCASE) from the Makefile?

I thought about that, however that would mean we have to document in the options that we're expecting a lower case version of the name, which would be something easy to miss as a developer. As a consequence, we'd have the transformation "just in case", and if we're having that anyway, we rather pass `APPLICATION_NAME` from the Makefile, which is more readable and in-line with the rest of the scripts rather than `APPLICATION_NAME_LOWERCASE`, since they'd have the same effect anyway.

> And I guess you could pass in $(APPLICATION_VERSION_DEBIAN) too?

I agree with that! I'll update the PR!